install/devsds/lib/etcd.sh

```diff
@@ -41,9 +41,9 @@ osds::etcd::download() {
 
 osds::etcd::install() {
     # validate before running
-    which etcd >/dev/null || {
-    osds::etcd::download
-    }
+    if [ ! -f "${OPT_DIR}/etcd-v${ETCD_VERSION}-linux-amd64/etcd" ]; then
+        osds::etcd::download
+    fi
 
```

Comment:
it's alse be fine if etcd is install on the system path

Reply:
A etcd latest version (3.4) will not work with installer if it is present in system path. We need etcd version 3.3

Reply:
ok